.infrastructure/helm-chart/todoapp/templates/app/configMap.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+  namespace: {{ .Values.common.namespace }}
+data:
+  {{- range $k, $v := .Values.config }}
+  {{ $k }}: {{ $v | quote }}
+  {{- end }}
```

Comment:
Ensure that `.Values.config` is defined in your `values.yaml` file. This is necessary for populating the ConfigMap with the correct data.